### PR TITLE
Add vice presidents

### PR DESCRIPTION
**Pull request by hayeselnut**

### CODE REVIEW

File: CONSTITUTION.rst [hunk 1]
```diff
@@ -359,6 +361,25 @@ The GEDI Officer
    #. Monitoring engagement and membership of students from non-majority demographics within the Club and provide regular updates to the Club Executive;
    #. Other relevant duties as required.
 
+Vice President (External)
+"""""""""""""""""""""
+
+#. The duties of the Vice President (External) shall be:
+
+   #. Supervise the external facing aspects of the society, especially the Careers and Events portfolios.
```
Comment (SpanishPear) [inline CONSTITUTION.rst:369]: Careers and Events portfolios do/should not exist in the constitution

File: CONSTITUTION.rst [hunk 2]
```diff
@@ -359,6 +361,25 @@ The GEDI Officer
...
+   #. Sustain and build external relationships and sponsors for CSESoc.
+   #. Facilitate culture and team relations within the society.
+   #. Other relevant duties as required.
+
+Vice President (Internal)
+"""""""""""""""""""""
+
+#. The duties of the Vice President (Internal) shall be:
+
+   #. Supervise the internal facing operations of the society, especially the Content and Development portfolios.
```
Comment (SpanishPear) [inline CONSTITUTION.rst:379]: Content/Development portfolios do/should not exist in the constitution

E.g. - dev likely wont exist in '23

**Review by SpanishPear — CHANGES_REQUESTED** [review]
see comments

**Comment by hayeselnut** [comment]
https://drive.google.com/file/d/1iXmM0ppUenm5CwZxacW2nKQPLqU9IMW2/view?fbclid=IwAR2lqduZ2pNdR0RI9VGfsCTcNBoEZpvvFbWMljXd0UV-NDyMauUkG08qYBY

**Comment by lting0804** [comment]
Clause 3.3.2. ii currently says at least 2 candidates for Co President and 1 for each other position

needs to be amended to say at least 2 for Vice President Development as well :)

**Comment by lting0804** [comment]
Approved at AGM

**Pull request merged**